### ISSUE TITLE
Give possibility to disembed resources and vendor dirs in other location than base dir

### ISSUE THREAD

**Issue by txthinking** [asticode/go-astilectron#78]
I know `resources` is created by bootstrap + bundler #42.
It does not matter on MacOS, but on Windows it will let user feel strange. (I don't try on Linux)
Is it possible:

* build in binary use go-bindata (may use more memory, but a option)
* put it in the User Home Directory and make it a hidden file

Thank you.

**Comment by Noofbiz** [comment]
On windows I had the msi put the folder in App Data folder. I used wix as this was before the bundler, but it worked okay for me.

**Comment by asticode** [comment]
Hey guys, I've pushed a change doing the following:

- the `Paths` object now has a `DataDirectory()` method
- it returns 
  - `%APPDATA%/<your app name>` if the `APPDATA` environment variable exists (windows)
  - the working directory for other cases
- the `vendor` directory is now installed in this `data directory`

This means the `vendor` directory is now in the APPDATA folder and "hidden" for the user.

Also, if you're using the [bootstrap](https://github.com/asticode/go-astilectron-bootstrap), I've pushed a change making sure the `resources` folder is also installed in this `data directory`.

I'll close this issue for now but do reopen it if it doesn't fix your issue.

Cheers
